IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -102,6 +102,8 @@ - (void)executeRequestWithCompletion:(MSIDRequestCompletionBlock)completionBlock
         return;
     }
     
+    [self.requestParameters updateAppRequestMetadata:nil];
```

Comment:
Will we need to do the same for broker requests? Possible to have it done somewhere globally?

Reply:
Refactored, no changes needed on the broker side now.